tools/python/boutiques/localExec.py

```diff
@@ -292,6 +293,11 @@ def execute(self, mount_strings):
             if launchDir is None:
                 launchDir = op.realpath('./')
             launchDir = op.realpath(launchDir)
+            # Get the container options
+            conOptsString = ""
+            if conOpts:
+                for opt in conOpts:
+                    conOptsString += opt + ' '
```

Comment:
In a perfect world, we could add some validation on what strings were passed, and that they're valid. But this requires options to be well documented by tools.